client/src/test/java/org/spine3/protobuf/TimestampsShould.java

```diff
@@ -48,14 +49,24 @@ public void have_private_constructor() {
         assertTrue(hasPrivateUtilityConstructor(Timestamps.class));
     }
 
+    @Test
+    public void do_not_throw_exception_if_timestamp_is_valid() {
```

Comment:
Following the Should paradigm, I would name this method `not_throw_exception...`.

In your case the test reads `Timestamps should do not throw ...`, which is does not make much sense in English.